runtime/gen/docs.tmpl

```diff
@@ -14,8 +14,7 @@ A quick note about colors.  When specifying colors, use a CSS-like
 hexdecimal color specification.  Pixlet supports "#rgb", "#rrggbb",
 "#rgba", and "#rrggbbaa" color specifications.
 
-{{range .}}{{$widgetName := .Name}}
-{{if .Documentation}}
+{{range .}}{{if .Documentation}}{{$widgetName := .Name}}
```

Comment:
As `AnimatedPositioned` is not documented yet, it resulted in generating an empty section.